### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheYahton**
a nice markdown table in Platform support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,17 +38,22 @@ For now, the priority for new platforms is:
 
 (PRs will be accepted for any platform, even if it does not follow the above priority.)
 
-✅: Present | ❌: Absent
- - AndroidNdk ❌
- - AppKit ✅ (Thanks to [Seo Sanghyeon](https://github.com/sanxiyn) and [lunixbochs](https://github.com/lunixbochs)!)
- - Orbital ✅
- - UiKit ❌
- - Wayland ✅ (Wayland support in winit is immature at the moment, so it might be wise to force X11 if you're using winit)
```
Comment (madsmtm) [inline README.md:46]: Pretty sure this comment is outdated, @kchibisov?

Comment (kchibisov) [reply README.md:46]: pretty much.

File: README.md [hunk 2]
```diff
@@ -38,17 +38,22 @@ For now, the priority for new platforms is:
...
- - WinRt ❌
- - Xcb ✅ 
- - Xlib ✅
+|  Platform ||
+|-----------|--|
+|Android NDK|❌|
+|   AppKit  |✅|
+|  Orbital  |✅|
+|    UIKit  |❌|
+|  Wayland  |❔|
```
Comment (notgull) [inline README.md:47]:
```suggestion
|  Wayland  |✅|
```

Wayland has pretty good support in `softbuffer`, at least I think so (paging @ids1024)

**Review by notgull — CHANGES_REQUESTED** [review]

**Review by notgull — APPROVED** [review]
Thanks! Please rebase on master to fix the CI issues.

**Pull request merged**